src/Xenial.Tasty/Protocols/SerialiTestCase.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Xenial.Delicious.Metadata;
+
+namespace Xenial.Delicious.Protocols
```

Comment:
We need a way more less noisy way to use the TestCases and Groups directly as protocol without copying and duplicating the APIs

Reply:
We should address this in a separate PR